ecs-agent/netlib/common_test.go

```diff
@@ -115,10 +115,10 @@ func getMultiNetNSMultiIfaceAWSVPCTestData(testTaskID string) (*ecsacs.Task, tas
 		ElasticNetworkInterfaces: enis,
 		Containers: []*ecsacs.Container{
 			{
-				NetworkInterfaceNames: []*string{aws.String(ifName1)},
+				NetworkInterfaceNames: []*string{aws.String(ifName2)},
```

Comment:
May I ask why did we swap the order here?

Reply:
The customer can specify the order of interfaces however they wish. This test ensures we do not make any assumptions in our code against that possibility.